Update docskit-mdx-partials.mdx (#76)
Platform-OS/docskit-landing: src/docs/03-case-studies/docskit-mdx-partials.mdx

---

> title: Single sourcing reusable content with DocsKit Partials

Reviewer: It reads as a tautology to me because "single sourcing" already implies that you are reusing content. Can you think a bit more on the title? 🤔 the keywords are single sourcing, content library, centralized, content blocks, reusing, but single sourcing and reusing are too much together.

If we want go with Occam's razor, it is "Single sourcing content with DocsKit Partials".

Author: Thank you! I edited it to be "Single sourcing content with DocsKit Partials".

slug: case-studies/docskit-mdx-partials
description: DocsKit Partials redefine content creation by offering a centralized solution to the challenge of maintaining consistency and efficiency in user guides and technical documentation.
hideToc: true
---
import bgImage from './assets/partials.svg';

<Card variant="hero" bgImage={bgImage}> 
# Single sourcing reusable content with DocsKit Partials
## Write once, use it everywhere.
</Card>

## Project overview

Single sourcing in technical writing is a methodology where content is created once and reused across multiple documents or platforms. This approach streamlines content management, ensuring consistency and efficiency, and simplifies updates in technical documentation.

> DocsKit enables single sourcing through Partials, reusable content blocks (like paragraphs, images, or chapters) managed centrally (in a single source) and embedded wherever needed within documentation.

Reviewer: ```suggestion
DocsKit enables single sourcing through Partials, which are reusable content blocks (they can be paragraphs, images, or even entire chapters). These building blocks are stored in a central library (a single source) and can be easily inserted wherever you need them in your documentation.
```

> DocsKit Partials enable writers to maintain a library of content snippets that can be updated once and reflected across all instances where they are used, significantly streamlining the process by avoiding repetitive updates, ensuring content consistency, reducing redundancy and enhancing accuracy throughout documentation.

Reviewer: ```suggestion
DocsKit Partials empower writers to maintain a library of content snippets that can be updated once and reflected across all instances where they are used, significantly streamlining the process by avoiding repetitive updates, ensuring content consistency, reducing redundancy and enhancing accuracy throughout documentation.
```

> DocsKit Partials enable writers to maintain a library of content snippets that can be updated once and reflected across all instances where they are used, significantly streamlining the process by avoiding repetitive updates, ensuring content consistency, reducing redundancy and enhancing accuracy throughout documentation.

Reviewer: the previous paragraph also started with _X enables Y_

<Card shadow variant="gradient">
## Summary

**Challenge:** Updating and reviewing repetitive content in documentation.  

> **Solutions:** Single sourcing, creating a centralized reusable content library.  

Reviewer: ```suggestion
**Solutions:** Simplifying the process of single sourcing by creating a centralized reusable content library.  
```

<Grid cols="3">
  <Card variant="feature" shadow icon={icon1}>
    #### Streamline content creation

>     Enable writers to compile a library of reusable content, facilitating single sourcing of paragraphs, images, or entire chapters to reduce the need to recreate the same content in multiple places and save time.

Reviewer: Another "enable". :)

Author: Thank you! "Enable" changed to "Allow"
